dev/tasks/docs/github.linux.yml

```
@@ -44,23 +44,18 @@ jobs:
           ref: {{ default_branch|default("master") }}
           path: crossbow
           fetch-depth: 1
-      - name: Move docs
+      - name: Prepare docs
         run: |
           # build files are created by the docker user
           sudo chown -R ${USER}: build
-          PR_DIR=crossbow/docs/pr_docs/{{ pr_number }}
-          mkdir -p $PR_DIR
-          rsync -a --delete build/docs/ $PR_DIR
-      - name: Push changes
+          # These are only used while generating the page
+          rm -rf build/docs/.doctrees
+      - name: Upload preview to S3
+        env: 
+          AWS_ACCESS_KEY_ID: {{ '${{ secrets.CROSSBOW_DOCS_AWS_ACCESS_KEY_ID }}' }}

```

I am probably missing something but wouldn't the following work?
```suggestion
          AWS_ACCESS_KEY_ID: ${{ secrets.CROSSBOW_DOCS_AWS_ACCESS_KEY_ID }}

```

---

No, the github "macro" `${{}}` needs to be wrapped in jinja `{{}}` otherwise jinja will complain that there is no var named secrets. It's a bit clunky but I don't know of a better way to prevent jinja from complaining.

---

@kszucs for insight

---

I see, it is a way of escaping the `{{ }}` on Jinja. I suppose you could use the following tags? I haven't tested so I might be wrong but I understand why you are doing it now: https://jinja.palletsprojects.com/en/3.1.x/templates/#escaping
```
{% raw %}
{% endraw %}
```

---

@assignUser Do you want to try @raulcd's suggestion?

---

I don't think it is necessary for this case as we only have the two insertions, I will keep it in mind for future applications though!